packages/settingeditor/src/settingseditor.tsx

```diff
@@ -0,0 +1,99 @@
+import { ISettingRegistry, Settings } from '@jupyterlab/settingregistry';
+import { IStateDB } from '@jupyterlab/statedb';
+import { ITranslator } from '@jupyterlab/translation';
+import { IFormComponentRegistry, ReactWidget } from '@jupyterlab/ui-components';
+import { PanelLayout, SplitPanel, Widget } from '@lumino/widgets';
+import React from 'react';
+import { PluginList } from './pluginlist';
+import { SettingsPanel } from './settingspanel';
+import { Message } from '@lumino/messaging';
+import { Dialog, showDialog } from '@jupyterlab/apputils';
+
+export class SimpleSettingsEditor extends Widget {
+  translator: any;
+  //   private _settings: any[];
+  //   private _settingsPanel: any;
+  private _list: any;
+  private _panel: SplitPanel;
```

Comment:
Could you place the attribute declaration at the bottom of the class to be consistent with JLab convention.